lib/internal/bootstrap/node.js

```diff
@@ -246,6 +247,60 @@ if (!config.noBrowserGlobals) {
     return performance;
   });
 
+  defineOperation(globalThis, 'prompt', function prompt(
+    message = 'prompt',
+    defaultValue
+  ) {
+    const { validateString } = require('internal/validators');
+    validateString(message);
+    if (defaultValue !== undefined) {
+      validateString(defaultValue);
+    }
+
+    defaultValue ??= null;
+    if (!process.stdin.isTTY) {
+      return null;
+    }
+
+    process.stdout.write(message + '\r\n');
+    const res = readFromStdInSync();
+    return res || defaultValue;
+
+    // Mimic implementation of `prompt` of deno
+    function readFromStdInSync() {
+      const fs = require('fs');
+      const LF = '\n'.charCodeAt(0);
```

Comment:
```suggestion
      const LF = StringPrototypeCharCodeAt('\n', 0);
```
Use primordials for consistency.